contributors/devel/development.md

```diff
@@ -117,6 +117,9 @@ Kubernetes maintains state in [`etcd`][etcd-latest], a distributed key store.
 
 Please [install it locally][etcd-install] to run local integration tests.
 
+### Development using local cluster
+After installing etcd, we recommend running a local cluster for development purpose. Refer this [tutorial](https://kubernetes-v1-4.github.io/docs/getting-started-guides/locally/) for more information.
```

Comment:
Format in shorter lines, please.

That doesn't seem like a very helpful link to me.

It immediately says, stop, go use minikube, and minikube cannot be used for development so, it is deceptive, if anything.

Possibly link to 
https://kubernetes-v1-4.github.io/docs/getting-started-guides/locally/#starting-the-cluster
or 
https://kubernetes-v1-4.github.io/docs/getting-started-guides/locally/#i-changed-kubernetes-code-how-do-i-run-it

That whole document should probably go away and and any mention of `hack/local-up-cluster.sh` should be mentioned inline.